dex/networks/eth/tokens.go

```diff
@@ -199,18 +215,20 @@ var Tokens = map[uint32]*Token{
 							// map.
 							// Average of 5 transfers: 51820
 							//   [65500 48400 48400 48400 48400]
-							Swap:      175_000,
-							SwapAdd:   115_000,
-							Redeem:    87_000,
-							RedeemAdd: 33_000,
-							Refund:    67_000,
-							Approve:   65_000,
-							Transfer:  70_000,
+							//
+							// Then buffered by about 130%...
+							Swap:      242_000, // actual ~187,880 -- https://goerli.etherscan.io/tx/0x352baccafa96bb09d5c118f8dcce26e34267beb8bcda9c026f8d5353abea50fd, verified on mainnet at 188,013 gas
+							SwapAdd:   146_400, // actual ~112,639 (300,519 for 2) -- https://goerli.etherscan.io/tx/0x97f9a1ed69883a6e701f37883ef74d79a709e0edfc4a45987fa659700663f40e
+							Redeem:    102_700, // actual ~79,012 -- https://goerli.etherscan.io/tx/0x96f007036b01eb2e44615dc67d3e99748bc133496187348b2af26834f46bfdc8, verified on mainnet at 79,113 gas
```

Comment:
I saw redeem gas of 83,850 while testing #2050.

Reply:
I bet that is because it was before the redeployment in https://github.com/decred/dcrdex/pull/2111
The contract didn't change much, but it was also a different solc version.  Not sure.

Reply:
Oh, when an account receives a token for the first time, I think that requires more gas to allocate storage space for the new non-zero balance, the next redeem will be cheaper (not referring to batches either).

I'm finding it's the same with token *transfer* too.  When an account receives USDC for the first time, it seems to be a 65,524 gas, but after that it's 48,424)